tests/unit/transformers/test_categorical.py

```diff
@@ -700,44 +705,6 @@ def test__get_value_add_noise_true(self, norm_mock):
         # Asserts
         assert result == 0.2745
 
-    def test__reverse_transform_array(self):
```

Comment:
`BaseTransformer.reverse_transform` specifies the input data should be a pandas DataFrame so I removed this test on a numpy array.

Reply:
Maybe you should change the data to be a pd.Series instead? Because this is the only test we have for the `_reverse_transform` in the FrequencyEncoder, so the logic may end up not being tested if you remove it.